### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by double22a**
basd  #480, add results of chunk_size=full

### CODE REVIEW

**Comment by double22a** [comment]
based on  #480

File: examples/multi_cn/s0/README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ thcs30 results
 |--------------------------|------|------|
 | attention decoder        | 16.68 | 17.47 |
```
Comment (pengzhendong) [inline examples/multi_cn/s0/README.md:50]: Could you align this table?

**Review by double22a — COMMENTED** [review]
align

**Review by pengzhendong — APPROVED** [review]
Good job

File: examples/multi_cn/s0/README.md (hunk 1 above)
Comment (double22a) [reply examples/multi_cn/s0/README.md:50]: align

**Pull request merged**